lib/index.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     'no-blur': require('./rules/no-blur'),
     'no-d-none': require('./rules/no-d-none'),
     'no-dataset': require('./rules/no-dataset'),
+    'no-generic-link-text': require('./rules/no-generic-link-text'),
```

Comment:
I want this to be prefixed so I can pattern match the code base specifically for accessibility rules. What should I do???

[Related Slack thread on prefixing eslint rules ](https://github.slack.com/archives/CMZ4DC9BL/p1658186503467939?thread_ts=1658175507.494509&cid=CMZ4DC9BL)(GitHub Staff only)

Reply:
I changed the name to `a11y-`

Reply:
I like the a11y!